### PR TITLE
Tooltip: remove name from live Region

### PR DESCRIPTION
**Pull request by jiabaowu**
Fixes: #11272

### CODE REVIEW

**Comment by scottgonzalez** [comment]
The test is outside of the main closure, but this needs to be rebased anyway since you were working on an outdated branch.

File: tests/unit/tooltip/tooltip_core.js [hunk 1]
```diff
@@ -220,3 +220,21 @@ asyncTest( "multiple active delegated tooltips", function() {
 });
 
 }( jQuery ) );
+
+// http://bugs.jqueryui.com/ticket/11272
+test ( "remove radio button name conflict between live Region and tooltip content", function () {
```
Comment (scottgonzalez) [inline tests/unit/tooltip/tooltip_core.js]: Let's rename to "remove conflicting attributes from live region"

Comment (scottgonzalez) [inline tests/unit/tooltip/tooltip_core.js]: extraneous space after `function`

File: tests/unit/tooltip/tooltip_core.js [hunk 2]
```diff
@@ -220,3 +220,21 @@ asyncTest( "multiple active delegated tooltips", function() {
...
+
+	var element = $( "<div id='content'>" +
+		"<input type='radio' name='hobby' checked='checked'><label>option 1</label>" +
+		"<input type='radio' name='hobby'><label>option 2</label>" +
+		"</div>" );
+	$( "#tooltipped1" )
+		.tooltip({
+			content: element,
+			open: function( event, ui ) {
+				equal( $( ".ui-helper-hidden-accessible [name]" ).length, 0, 'no name attr within liveRegion');
```
Comment (scottgonzalez) [inline tests/unit/tooltip/tooltip_core.js]: Split the message on to a separate line and use double quotes.

Comment (scottgonzalez) [inline tests/unit/tooltip/tooltip_core.js]: Let's add an assertion for `id` as well

Comment (scottgonzalez) [inline tests/unit/tooltip/tooltip_core.js]: "no name attr within liveRegion" -> "no name attributes within live region"

**Comment by jiabaowu** [comment]
Thanks for all the comments scott! :D hopefully I am making all the mistakes so I wont have to later.

**Comment by jiabaowu** [comment]
OK hopefully with those changes its better, thank you for closing the other pull request, very nice workflow here :D haha

**Comment by scottgonzalez** [comment]
Thanks, squashed and landed with a bit more cleanup.

**Pull request closed without merging**